draft-bbs-signatures.md

```diff
@@ -157,6 +157,9 @@ a || b
 I \\ J
 : For sets I and J, denotes the difference of the two sets i.e., all the elements of I that do not appear in J, in the same order as they were in I.
 
+X\[a..b\]
+: Denotes a slice of the array `X` containing all elements from index `a` until `b`. Note when this syntax is applied to an octet string, each element in the array `X` is assumed to be a single byte.
```

Comment:
Should probably clarify if the range is inclusive or exclusive

Reply:
@andrewwhitehead good point, please check the updated text